PWGMM/Mult/Tasks/dndeta-hi.cxx

```diff
@@ -71,23 +81,24 @@ static constexpr TrackSelectionFlags::flagtype trackSelectionTPC =
 static constexpr TrackSelectionFlags::flagtype trackSelectionDCA =
   TrackSelectionFlags::kDCAz | TrackSelectionFlags::kDCAxy;
 
-using LabeledTracks = soa::Join<aod::Tracks, aod::McTrackLabels>;
-
 struct MultiplicityCounter {
   Service<TDatabasePDG> pdg;
 
   Configurable<float> estimatorEta{"estimatorEta", 1.0, "eta range for INEL>0 sample definition"};
   Configurable<bool> useEvSel{"useEvSel", true, "use event selection"};
   Configurable<bool> isMC{"isMC", false, "check if MC"};
+
   Service<ccdb::BasicCCDBManager> ccdb;
   Configurable<std::string> path{"ccdb-path", "Users/s/sherrman/My/Object", "base path to the ccdb object"};
   Configurable<std::string> url{"ccdb-url", "http://alice-ccdb.cern.ch", "url of the ccdb repository"};
-  Configurable<int64_t> nolaterthan{"ccdb-no-later-than", std::chrono::duration_cast<std::chrono::milliseconds>(std::chrono::system_clock::now().time_since_epoch()).count(), "latest acceptable timestamp of creation for the object"};
 
-  Configurable<int> MultBin{"MultBin", 100, "# of Multiplicity bin"};
-  Configurable<double> MultMin{"MultMin", 0, "minimum of Multiplicity bin"};
-  Configurable<double> MultMax{"MultMax", 10000, "maximum of Multiplicity bin"};
-  AxisSpec MultAxis = {MultBin, MultMin, MultMax};
+  Configurable<int64_t> nolaterthan{"ccdb-no-later-than", std::chrono::duration_cast<std::chrono::milliseconds>(std::chrono::system_clock::now().time_since_epoch()).count(), "latest acceptable timestamp of creation for the object"};
```

Comment:
This configurable and the three after it are not used anywhere in the code, why do you need them?

Reply:
I don't know how to use the configuration of "ccdb". when I copy some code for making dndeta-hi.cxx, maybe this part is copied too. Can I skip the "ccdb" configuration?

Reply:
You do not need to configure ccdb if you do not query it directly from your task. The configurables in your task are just special variables that can receive values externally, they do not configure anything unless you use them explicitly. You can safely remove these.